### PR TITLE
Replaced description by data format of various parser plugins #3087

### PR DESCRIPTION
**Pull request by joachimmetz**
Replaced description by data format of various parser plugins #3087

### CODE REVIEW

File: plaso/parsers/syslog_plugins/cron.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ class CronSyslogPlugin(interface.SyslogPlugin):
   """A syslog plugin for parsing cron messages."""
 
   NAME = 'cron'
-  DESCRIPTION = 'Parser for syslog cron messages.'
+  DATA_FORMAT = 'Cron syslog file'
```
Comment (rgayon) [inline plaso/parsers/syslog_plugins/cron.py:35]: Isn't this rather for "cron-relevant lines in a syslog file" ?

File: plaso/parsers/syslog_plugins/ssh.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ class SSHSyslogPlugin(interface.SyslogPlugin):
   """A plugin for creating events from syslog message produced by SSH."""
 
   NAME = 'ssh'
-  DESCRIPTION = 'Parser for SSH syslog entries.'
+  DATA_FORMAT = 'SSH syslog file'
```
Comment (rgayon) [inline plaso/parsers/syslog_plugins/ssh.py:61]: Same here

File: plaso/parsers/syslog_plugins/cron.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/parsers/syslog_plugins/cron.py]: ack, changed file into line

File: plaso/parsers/syslog_plugins/ssh.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/parsers/syslog_plugins/ssh.py]: ack, changed file into line

**Comment by joachimmetz** [comment]
@rgayon PTAL

**Review by rgayon — APPROVED** [review]

**Pull request merged**